backend/src/apiserver/client/sql.go

```diff
@@ -15,13 +15,14 @@
 package client
 
 import (
+	"bytes"
 	"fmt"
 
 	"github.com/go-sql-driver/mysql"
 )
 
-func CreateMySQLConfig(user, password string, mysqlServiceHost string,
-	mysqlServicePort string, dbName string, mysqlGroupConcatMaxLen string, mysqlExtraParams map[string]string,
+func CreateMySQLConfig(user, password, mysqlServiceHost, mysqlServicePort,
```

Comment:
I don't have a strong preference, I'm doing this solely for style purpose. Since all these parameters are of string type, it feels more succinct to declare their type together.

Reply:
Makes sense, I am just curious about this change, thank you for the explanation!